examples/web/assets/js/app.js

```diff
@@ -68,7 +71,7 @@ openWindowButton.onclick = function () {
 			resizable: true,
 			x: 10, y: 10,
 			width: 850, height: 500,
-			minWidth: 200, minHeight: 100, maxWidth: 800, maxHeight: 600,
+			minWidth: 200, minHeight: 100, maxWidth: 850, maxHeight: 600,
```

Comment:
When testing window move, Electron would change the window width during a call to setBounds when only moving a window.  This is because the initial width was changed to be greater than max width and Electron would "fix" that.